__tests__/index.test.ts

```diff
@@ -34,3 +34,7 @@ test("doubleStruck", () => {
 test("circle", () => {
   expect(circle(origin)).toEqual("ⒶⒷⒸⒹⒺⒻⒼⒽⒾⒿⓀⓁⓂⓃⓄⓅⓆⓇⓈⓉⓊⓋⓌⓍⓎⓏⓐⓑⓒⓓⓔⓕⓖⓗⓘⓙⓚⓛⓜⓝⓞⓟⓠⓡⓢⓣⓤⓥⓦⓧⓨⓩ⓪①②③④⑤⑥⑦⑧⑨我爱你")
 })
+
+test("small", () => {
+  expect(small(origin)).toEqual("🇦🇧🇨🇩🇪🇫🇬🇭🇮🇯🇰🇱🇲🇳🇴🇵🇶🇷🇸🇹🇺🇻🇼🇽🇾🇿🇦🇧🇨🇩🇪🇫🇬🇭🇮🇯🇰🇱🇲🇳🇴🇵🇶🇷🇸🇹🇺🇻🇼🇽🇾🇿0123456789我爱你")  // god
```

Comment:
It seems very interesting that some characters join together then turn into national flag.

Reply:
National Flag?  😮